content/en/docs/ops/ambient/architecture/index.md

```diff
@@ -13,4 +13,77 @@ This page is under construction.
 
 ## Traffic routing
 
+In ambient mode, workloads can fall into 3 categories:
+* Uncaptured: this is a standard pod without any mesh features enabled
+* Captured: this is a pod that has traffic intercepted by `ztunnel`. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
+* Waypoint enabled: this is a pod that is "Captured" *and* has a waypoint proxy deployed.
+  A waypoint will, by default, apply to all pods in the same namespace.
+  It can optionally be set to apply to only a specific namespace with the `istio.io/for-service-account` annotation on the `Gateway`.
+  If there is both a namespace waypoint and service account waypoint, the service account waypoint takes precedence.
+
+Depending on which category a workload is in, the request path will be different.
+
+### Ztunnel routing
+
+#### Outbound
+
+When a captured pod makes an outbound request, it will be transparently redirected to `ztunnel` which will determine where and how to forward the request.
+In general, the traffic routing behaves just like Kubernetes default traffic routing.
+Requests to a `Service` will be sent to an endpoint within the `Service`, and requests directly to a `Pod` IP would be go directly to that `Pod` IP.
+
+However, depending on the destination's capabilities, different behavior will occur.
+If the destination is also captured, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted HBONE tunnel.
+If the destination has a waypoint proxy, in addition to being upgraded to HBONE, the request will instead be forwarded to that waypoint.
+
+Note that in the case of a request to a `Service`, a specific endpoint will be selected to determine if it has a waypoint.
+However, if it *has* a waypoint, the request will be sent with a target destination of the `Service`, not the selected endpoint.
+This allows the waypoint to apply service-oriented policies to the traffic.
+In the rare case that a `Service` has a mix of waypoint enabled and non-enabled endpoints, this would mean that some of our requests would be sent to a waypoint while others would not.
+
+#### Inbound
+
+When a captured pod receives an inbound request, it will be transparently redirected to `ztunnel`.
+When `ztunnel` receives the request, it will apply Authorization Policies and forward the request only if the request meets the policies.
+
+A pod can receive HBONE traffic or plaintext traffic.
+By default, both will be accepted by `ztunnel`.
+However, when evaluating Authorization Policies for plaintext traffic, the peer identity would be unset.
```

Comment:
This seems unclear: `peer identity would be unset`? Is it "not set" or does it get unset?

Reply:
Yeah wasn't sure how to word this.. what I mean is if its plaintext, there is no peer identity. So any policy matching on an identity would fail. So to deny plaintext, youcan make a policy that only allows some identity